### PR TITLE
Support GitLab MR comment bodies (optionally)

### LINKED ISSUE #499: Support Gitlab comments in MR Comment events
**Is your feature request related to a problem? Please describe.**
> Cause now I still need to jump to gitlab to review messages

**Describe the solution you'd like**
> On gitlab bridge can we put gitlab comment body to the element chat?

**Describe alternatives you've considered**
Using webhooks

**Additional context**
This feature is asked by an element onprem customer.

### PR DESCRIPTION
**Pull request by Half-Shot**
Fixes #499

### CODE REVIEW

File: docs/usage/room_configuration/gitlab_project.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ This connection supports a few options which can be defined in the room state:
 |ignoreHooks|Choose to exclude notifications for some event types|Array of: [Supported event types](#supported-event-types) |*empty*|
 |commandPrefix|Choose the prefix to use when sending commands to the bot|A string, ideally starts with "!"|`!gh`|
 |pushTagsRegex|Only mention pushed tags which match this regex|Regex string|*empty*|
-|prDiff|Show a diff in the room when a PR is created, subject to limits|`{enabled: boolean, maxLines: number}`|`{enabled: false}`|
```
Comment (jaller94) [inline docs/usage/room_configuration/gitlab_project.md:29]: Why was this removed?
Seems unrelated.

File: src/Connections/GitlabRepo.ts [hunk 2]
```diff
@@ -596,9 +602,9 @@ ${data.description}`;
             this.debounceMRComments.delete(uniqueId);
             const orgRepoName = project.path_with_namespace;
             let comments = '';
-            if (result.comments === 1) {
+            if (result.commentCount === 1) {
                 comments = ' with one comment';
-            } else if (result.comments > 1) {
+            } else if (result.commentCount > 1) {
                 comments = ` with ${result.comments} comments`;
```
Comment (jaller94) [inline src/Connections/GitlabRepo.ts:608]:
```suggestion
                comments = ` with ${result.commentCount} comments`;
```

File: docs/usage/room_configuration/gitlab_project.md [hunk 3]
```diff
@@ -26,9 +26,9 @@ This connection supports a few options which can be defined in the room state:
...
 |includingLabels|Only notify on issues matching these label names|Array of: String matching a label name|*empty*|
 |excludingLabels|Never notify on issues matching these label names|Array of: String matching a label name|*empty*|
+|includeCommentBody|Include the body of a comment when notifying on merge reqeuests|Boolean|false|
```
Comment (jaller94) [inline docs/usage/room_configuration/gitlab_project.md:31]:
```suggestion
|includeCommentBody|Include the body of a comment when notifying on merge requests|Boolean|false|
```

**Review by jaller94 — CHANGES_REQUESTED** [review]
Typo and wrong variable, other than that

~Please avoid weird union types.~ Looks good to me.

File: docs/usage/room_configuration/gitlab_project.md (hunk 1 above)
Comment (Half-Shot) [reply docs/usage/room_configuration/gitlab_project.md:29]: Option never existed, copied from GitHub

**Review by jaller94 — APPROVED** [review]
Looks good to me.
